src/main/java/com/setup/Setup.java

```diff
@@ -0,0 +1,23 @@
+package com.setup;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Setup {
```

Comment:
@PavelNejinskiy how about introducing also `Extensions` class, that will encapsulate all extension related logic?
Please also rename getPath to just `path` and `getExtension` to `extensions` . `addExtensionList` functionality should go to`Extensions` class like `add`